src/index.ts

```diff
@@ -3,15 +3,16 @@
 import { Command } from "commander";
 
 import { logs, restart, shutdown, start, status } from "./commands";
+import { startDaemon } from "./commands/start-daemon.command";
 
 const program = new Command();
 
 program
-    .command("start [pmConfigFilePath]")
+    .command("start")
```

Comment:
How does the config file get loaded now?

Reply:
I just remembered that there was a discussion about that. I think we decided to not hardcode it so the file could be named differently. Nonetheless I prefer it this way anyway.